src/main/java/carracinggame/domain/CarNames.java

```diff
@@ -0,0 +1,24 @@
+package carracinggame.domain;
+
+public class CarNames {
+    private static final String REGEX = ",+[\\s\\t],+|^[\\s\\t]*,.*|.*,[\\s\\t]*$";
```

Comment:
```suggestion
public class CarNames {

    private static final String REGEX = ",+[\\s\\t],+|^[\\s\\t]*,.*|.*,[\\s\\t]*$";
```

다른 클래스들은 첫 라인을 띄우는 컨벤션을 유지하셨는데 이 클래스는 아니네요. 

일관성 있는 컨벤션이 중요하다 생각해요.

그 이유는 읽는 사람이 쉽게 이해할 수 있기 때문이에요, 눈에 익숙해지기도 하고!

Reply:
수정했습니다.